src/decorators/before.ts

```diff
@@ -1,19 +1,20 @@
-import { Handler, ErrorRequestHandler } from 'express';
+import { ErrorRequestHandler, Handler } from 'express';
 import { MetadataKey } from '../constants/metadata-key';
+
+import { TMiddlewareDefinition } from '../types/middleware-definition';
 import { isClassDecorator } from '../utils/is-class-decorator';
 import { isPropertyDecorator } from '../utils/is-property-decorator';
-import { TMiddlewareDefinition } from '../types/middleware-definition';
 
 export function before(middleware: Handler | ErrorRequestHandler): any {
-  return function(target: any, key?: string, descriptor?: PropertyDescriptor) {
+  return function (target: any, key: string, descriptor: PropertyDescriptor) {
```

Comment:
@SylvainEstevez I tried to figure out the proper typings for this, and what I found was key and descriptor never having the `optional?` modifier.

Reply:
Ah perfect. Then I guess I can just do `key as string` 👍